src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java

```diff
@@ -35,8 +35,8 @@ public static void main(String[] args) {
      * @param depositAnnualPercentage ежегодный простой процент
      */
     static float getTotalDepositAmount(int depositAmount, int depositYears, int depositAnnualPercentage) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+        float totalAmount = depositAmount+((float)(depositAmount*depositAnnualPercentage) / 100) * depositYears;
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"